packages/molly/src/routes/+server.ts

```diff
@@ -1,6 +1,6 @@
-// import { OPENAI_TOKEN_LIMIT, OPENAI_API_KEY } from "$env/static/private";
-// import MollyAI from "$lib/server/openai";
+import { OPENAI_TOKEN_LIMIT, OPENAI_API_KEY } from '$env/static/private';
+import MollyAI from '$lib/server/openai';
 
-// const molly = new MollyAI(OPENAI_API_KEY, Number(OPENAI_TOKEN_LIMIT));
+const molly = new MollyAI(OPENAI_API_KEY, Number(OPENAI_TOKEN_LIMIT));
 
-// export const GET = molly.handlers.chatCompletion.GET;
\ No newline at end of file
+export const GET = molly.handlers.chatCompletion.GET;
```

Comment:
```suggestion
export const POST = molly.handlers.chatCompletion.POST;
```

Reply:
Sorry for my oversight! Thanks for the suggestion :-)